analyze.go

```diff
@@ -737,6 +737,33 @@ var directives = map[string][]uint{
 	"http2_recv_timeout": {
 		ngxHTTPMainConf | ngxHTTPSrvConf | ngxConfTake1,
 	},
+	"http3": {
+		ngxHTTPMainConf | ngxHTTPSrvConf | ngxConfFlag,
+	},
+	"http3_hq": {
+		ngxHTTPMainConf | ngxHTTPSrvConf | ngxConfFlag,
+	},
+	"http3_max_concurrent_streams": {
+		ngxHTTPMainConf | ngxHTTPSrvConf | ngxConfTake1,
+	},
+	"http3_stream_buffer_size": {
+		ngxHTTPMainConf | ngxHTTPSrvConf | ngxConfTake1,
+	},
+	"quic_active_connection_id_limit": {
+		ngxHTTPMainConf | ngxHTTPSrvConf | ngxConfTake1,
+	},
+	"quic_bpf": {
+		ngxMainConf | ngxDirectConf | ngxConfFlag,
+	},
+	"quic_gso": {
```

Comment:
nit: I think these directives are alphabetically sorted? Might make sense to stick to the existing pattern.

Reply:
You're right! Fixed  the order